public/app/core/components/ForgottenPassword/ForgottenPassword.tsx

```diff
@@ -21,6 +20,7 @@ const paragraphStyles = (theme: GrafanaTheme) => css`
 export const ForgottenPassword: FC = () => {
   const [emailSent, setEmailSent] = useState(false);
   const styles = useStyles(paragraphStyles);
+  const loginHref = `${config.appSubUrl}/login`;
```

Comment:
Hmm, you should use `getConfig()` instead

Reply:
Commit on 4th Sept used config instead of get config. https://github.com/grafana/grafana/commit/a54df0c3cd37f79655a70d16a66a8a631d6e5304#diff-8ed33df727294c29ebfba45e47faba53

Let me know if you want me to change back to getConfig

Reply:
Yeah, I saw that change. If you can change it, but not super important. It much more important to get a backend specific code review.